addition of mysql-WP example

amygdala opened the pull request:
Brendan, you looked at a version of this earlier.  Does it make sense to include in the examples dir?

amygdala committed:
addition of mysql-wordpress-pd example

brendandburns commented:
Meta comment, before I do the main review:  can we make this be: "examples/mysql-wordpress-pd" instead of "WP" for a more readable description of what it is?

brendandburns commented on examples/mysql-WP-pd/README.md:
```diff
@@ -0,0 +1,318 @@
+
+Next, grab the Kubernetes [release binary](https://github.com/GoogleCloudPlatform/kubernetes/releases). 
+
+Then, start up a Kubernetes cluster as [described here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/gce.md).
```
Let's use the new:

```wget -q -O - https://get.k8s.io | sh``` or
```curl -sS https://get.k8s.io | sh```

amygdala replied:
Done.

brendandburns commented on examples/mysql-WP-pd/README.md:
```diff
@@ -0,0 +1,318 @@
+            mountPath: /var/www/html
+        env:
+          - name: WORDPRESS_DB_PASSWORD
+            # change this
```
Maybe note that this needs to match the password filled in previously?

amygdala replied:
Done.

brendandburns commented on examples/mysql-WP-pd/README.md:
```diff
@@ -0,0 +1,318 @@
+apiVersion: v1beta1
+id: wpfrontend
+# the port that this service should serve on
+port: 3000
```
We should use port 80 here.

amygdala replied:
Done.

brendandburns commented on examples/mysql-WP-pd/README.md:
````diff
@@ -0,0 +1,318 @@
+```
+
+You should see the familiar WordPress init page.
+
````
It might be nice for adding instructions to add a replication controller for the wordpress and sizing up from one instance to multiple instances.

amygdala replied:
I considered this, but I believe it will cause trouble due to the way "local" volume storage is used for images and plugins.  The replicas can't multiply write to the same PD.

amygdala commented:
Thanks. I've renamed the subdir and addressed your comments.  (With the exception of the replicated pods suggestion-- I think that is problematic. See my note on that).  I rebased, which I hope is useful rather than annoying, so it's just one commit.

satnam6502 commented:
What's the status of this PR? Thanks.

amygdala commented:
It's good to go IMO (addressed the comments).

amygdala commented:
Brendan, are we okay to pull this in?

brendandburns commented:
Sorry for the delay.
